.github/workflows/build.yml

```diff
@@ -82,11 +82,11 @@ jobs:
                 if: matrix.composer-mode == 'update'
                 env:
                     SYMFONY_REQUIRE: ${{ matrix.symfony-version }}.*
-                run: composer update
+                run: composer update ${{ matrix.php == '8.3' && '--ignore-platform-req=php+' || '' }}
 
             -   name: Install lowest dependencies
                 if: matrix.composer-mode == 'lowest'
-                run: composer update --prefer-lowest
+                run: composer update --prefer-lowest ${{ matrix.php == '8.3' && '--ignore-platform-req=php+' || '' }}
```

Comment:
this could be removed, as we don't have a lowest job on PHP 8.3 anyway

Reply:
Good catch. Fixed.